lib/modules/@apostrophecms/utils/src/apos/always/utils.js

```diff
@@ -264,4 +264,8 @@
     }
     return window.location.host === matches[2];
   };
+
+  apos.utils.generateId = function() {
```

Comment:
We should not be adding anything to the "always" scene if we only need it in the "user" scene. You can import the cuid module and use that directly.